apps/trustlab/.env.template

```diff
@@ -0,0 +1,28 @@
+#Next
+NEXT_PUBLIC_SERVER_URL=
```

Comment:
Why do we need this? Meaning will it ever be different from `NEXT_PUBLIC_APP_URL`?